packages/dds/merge-tree/src/collections.ts

```diff
@@ -1156,6 +1156,7 @@ export interface IInterval {
     compare(b: IInterval): number;
     compareStart(b: IInterval): number;
     compareEnd(b: IInterval): number;
+    modify(label: string, start: number, end: number);
```

Comment:
what's label? is that id?

Reply:
No, it's something that createSequenceInterval needs.